### PR TITLE
updated references of base plugins to netcommon 

### PR DESCRIPTION
**Pull request by KB-perByte**
##### SUMMARY

Minimum required ansible.netcommon version is 2.5.1
Updated base plugin references to ansible.netcommon

##### ISSUE TYPE

- Feature Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by KB-perByte** [comment]
recheck

**Comment by Qalthos** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

File: changelogs/fragments/netcommon_ref_update.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+major_changes:
+  - "Minimum required ansible.netcommon version is 2.5.0."
```
Comment (Qalthos) [inline changelogs/fragments/netcommon_ref_update.yaml:3]:
```suggestion
  - "Minimum required ansible.netcommon version is 2.5.1."
```
I don't see this enforced?

**Review by Qalthos — CHANGES_REQUESTED** [review]

**Review by Qalthos — APPROVED** [review]

**Pull request merged**